testutil/testkeyring/gen_accounts/gen.go

```diff
@@ -0,0 +1,91 @@
+//go:build ignore
+
+package main
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/pokt-network/poktroll/testutil/testkeyring"
+
+	"github.com/pokt-network/poktroll/app"
+)
+
+var (
+	flagOut           string
+	flagAccountsLimit int
+	defaultOutPath    = "accounts_table.go"
+)
+
+func init() {
+	flag.StringVar(&flagOut, "out", defaultOutPath, "the path to the generated file.")
+	flag.IntVar(&flagAccountsLimit, "limit", 100, "the number of accounts to generate.")
+}
+
+func main() {
+	flag.Parse()
+
+	kr := keyring.NewInMemory(app.MakeEncodingConfig().Marshaler)
+
+	preGeneratedAccountsGobHexLines := make([]string, flagAccountsLimit)
```

Comment:
Two questions about this naming:

1. The serialized value returned `hex`, so I think we can replace `HexLines` with `Hex`
2. We do `JSON` marshalling, so I believe we can remove `Gob`